nx.json

```diff
@@ -0,0 +1,46 @@
+{
+  "extends": "nx/presets/npm.json",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "test",
+          "lint",
+          "package",
+          "prepare"
+        ],
+        "accessToken": "${NX_CLOUD_ACCESS_TOKEN}"
```

Comment:
is this env variable required to interact on local machines? Do we as developers interact with nx on local machines or only in CI?

Reply:
we need to add that `NX_CLOUD_ACCESS_TOKEN` into `nx-cloud.env` file in our local machines that have cloud caching functionalities and for ci we need to add to env vars.

Reply:
is it necessary though to add it or only to have cloud caching?

Reply:
as we talked this is just to have cloud caching and if we dont provide nothing changes except the cloud caching will not work. but still we can local nx caching mechanism